### PR TITLE
Online access

### PR DESCRIPTION
**Pull request by Healthedata1**
offline_access Ballot#3, part of Ballot#44, Ballot#54, Ballot#63, Ballot#74, Ballot#119, Ballot#147, Ballot#202
#8,#195, #144, #123, #79,#67, #58, #48

online_access
Ballot#123, Ballot#75, part of Ballot#54, Ballot#50, part of Ballot#44
#127, #80, #58, #54, #48

### CODE REVIEW

File: spec/index.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+from the token response---
```
Comment (jmandel) [inline spec/index.md]: This looks like a stray line?

**Comment by Healthedata1** [comment]
yes see offline_access Ballot#3, part of Ballot#44, Ballot#54, Ballot#63, ... for the changes and the last change basecd on  Isaac's feedback.

**Comment by jmandel** [comment]
This PR links to `Ballot#44 #40` which was separately addressed by PR https://github.com/HL7/smart-app-launch/pull/241; is is really supposed to link to `Ballot#44 #40`?

Also, I'm only seeing a one-word change in this PR. If the changes are being implemented somewhere else, can you link to that? I don't see an open PR about offline access (I thought we closed it so we could make all of the required changes here instead.)

**Comment by Healthedata1** [comment]
look at the changes above isaacs comments abover  after which  I changed the line.   Also I don't know how to edit the commit text to change #44 to ballot#44

**Comment by Healthedata1** [comment]
Should I just scrap this one and redo?

**Comment by jmandel** [comment]
No, let me take a look outside of the GH view; it might just be  doing a poor job of showing changes.

**Comment by jmandel** [comment]
So the issues is that the https://github.com/HL7/smart-app-launch/pull/228/commits/e4e3d1c8a114de2f6c5b483f6f7dcffd1c9fe996 reversed all of the language you pulled in from https://github.com/HL7/smart-app-launch/pull/227. So you pulled them in and then overwrote them :/

**Comment by Healthedata1** [comment]
OK ... I'll close that and redo the pr

Eric M Haas, DVM, MS
Health eData Inc
1183 Mountain View Circle, Napa, 94558
707.227.2608|Skype: haas.eric1
ehaas@healthedatainc.com

**Pull request closed without merging**